Unity-Technologies/com.unity.multiplayer.samples.coop
# fix: handling of host disconnect

### Description (*)

This PR makes it so when the host gets disconnected by the API, it will be properly sent back to the main menu.
### Related Pull Requests

### Issue Number(s) (*)

Fixes issue(s): [MTT-2367](https://jira.unity3d.com/browse/MTT-2367)
### Manual testing scenarios

1. Start hosting a game using photon relay
2. When in-game, pull your network cable or disconnect from wifi
3. See that after a time, Photon will disconnect you, and BossRoom will send you back to main menu
### Questions or comments

### Contribution checklist
